src/coreclr/pal/inc/unixasmmacrosriscv64.inc

```diff
@@ -256,7 +256,11 @@ C_FUNC(\Name\()_End):
 
     __PWTB_FloatArgumentRegisters = \extraLocals
 
-    .if ((__PWTB_FloatArgumentRegisters % 16) != 0)
+    // Note, stack (see __PWTB_StackAlloc variable) must be 16 byte aligned,
+    // SIZEOF__FloatArgumentRegisters (0x40) is 16 byte aligned, that mean initial
+    // __PWTB_FloatArgumentRegisters value must be not 16 byte aligned and
+    // after add (120 + 64) provide 16 byte aligned result.
+    .if ((__PWTB_FloatArgumentRegisters % 16) == 0)
```

Comment:
It looks good to me. But only with this comment it is hard for me to understand.
I am not sure if it is better or not. This is my suggestion. It uses 8 bytes for padding.  Just have a look. Thanks very much.
```
--- a/src/coreclr/pal/inc/unixasmmacrosriscv64.inc
+++ b/src/coreclr/pal/inc/unixasmmacrosriscv64.inc
@@ -266,12 +266,11 @@ C_FUNC(\Name\()_End):
         __PWTB_TransitionBlock = __PWTB_TransitionBlock + SIZEOF__FloatArgumentRegisters
     .endif
 
-
     __PWTB_CalleeSavedRegisters = __PWTB_TransitionBlock
-    __PWTB_ArgumentRegisters = __PWTB_TransitionBlock + 120
+    __PWTB_ArgumentRegisters = __PWTB_TransitionBlock + 128
 
-    // Including fp, ra, s1-s11, tp, gp, and (a0-a7)arguments. (1+1+11+1+1)*8 + 8*8.
-    __PWTB_StackAlloc = __PWTB_TransitionBlock + 120 + 64
+    // Including fp, ra, s1-s11, tp, gp, padding and (a0-a7)arguments. (1+1+11+1+1+1)*8 + 8*8.
+    __PWTB_StackAlloc = __PWTB_TransitionBlock + 128 + 64
     PROLOG_STACK_ALLOC __PWTB_StackAlloc
     PROLOG_SAVE_REG_PAIR  fp, ra, __PWTB_CalleeSavedRegisters, 1
```

Reply:
In case `extraLocals` argument is not 16 byte aligned, with my changes we will use less stack memory (no extra +8 and again +8 for padding).

Reply:
Yes, I knew. Thank you!